compiler/passes/docs.cpp

```diff
@@ -17,24 +17,25 @@
  * limitations under the License.
  */
 
-#include <map>
-#include <iostream>
 #include <fstream>
+#include <iostream>
 #include <iterator>
-#include <sstream>
 #include <sys/stat.h>
 #include <sys/types.h>
+#include <sstream>
```

Comment:
Was this alphabetizing the includes?  If so, sstream definitely comes before sys/stat.h